source/concurrent/Promise.ooc

```diff
@@ -15,20 +15,23 @@ _PromiseState: enum {
 	Cancelled
 }
 
-Promise: abstract class {
+_Synchronizer: abstract class {
```

Comment:
Yeah, it bothered me that `Promise` and `Future` didn't have a base class in common. This is better.